kolibri/utils/cli.py

```diff
@@ -252,16 +261,12 @@ def __init__(self, settings, pythonpath):
         self.pythonpath = pythonpath
 
 
-def _setup_django(debug):
+def _setup_django():
     """
     Do our django setup - separated from initialize to reduce complexity.
     """
     try:
         django.setup()
-        if debug:
-            from django.conf import settings
-
-            settings.DEBUG = True
```

Comment:
NB! As noted in the description, this PR fixes the above behavior: Before, it would set `settings.DEBUG = True` **_too late_**, i.e. after it got used by various settings. Now it's being set directly from `options.conf`

I'm not sure if I think it's great that using `--debug` alters Django settings: I have been thinking more about scenarios for using it to debug the production environments, giving instructions to users to run with `--debug` or add `KOLIBRI_DEBUG` in their environment.

Django's DEBUG has stronger semantics as a development environment flag, for instance it switches on template debugging now. In this case, it doesn't enable loading and serving static files from app folders, though.. so that's nice... but perhaps also unexpected!

`settings.DEBUG=True` mixes development environment stuff into the picture, which could be disturbing and perhaps create unwanted troubleshooting issues... you all know how it is, when suddenly some unexpected development thing is enabled and vice versa :)

I'll try to inquire later in the 0.14 process to see what people think.